desc/compute/utils.py

```diff
@@ -407,6 +413,13 @@ def get_transforms(keys, obj, grid, jitable=False, **kwargs):
     return transforms
 
 
+def has_data_dependencies(parameterization, qty, data, axis=False):
```

Comment:
this just removes some unneeded checks on transform and grid during the recursion?